### PR TITLE
[PUB-3000] Replace user features by orgs in composer

### PR DESCRIPTION
**Pull request by anafilipadealmeida**
## Description
- Replace user features by orgs in composer

## Context & Notes
https://buffer.atlassian.net/browse/PUB-3000

## Checklist

-   [ ] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] I have tested different user stories. 
-   [ ] I have identified similar or related features and tested they are still working.
-   [ ] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [ ] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

**Comment by anafilipadealmeida** [comment]
Ignore the tests failing, as soon as I merge https://github.com/bufferapp/buffer-publish/pull/1567 and pull the changes here, it will be all good!

File: packages/composer/composer/components/ShopgridComposerBar.jsx [hunk 1]
```diff
@@ -65,7 +64,7 @@ class ShopgridComposerBar extends React.Component {
     return (
       hasInstagramSelected &&
       isInstagram &&
-      ((hasShopgridFlip && isBusinessUser) || isShopgridContributor)
+      (hasShopgridFlip || isShopgridContributor)
```
Comment (albennett) [inline packages/composer/composer/components/ShopgridComposerBar.jsx:67]: Do we remove the feature flip if the user downgrades?

Comment (albennett) [reply packages/composer/composer/components/ShopgridComposerBar.jsx:67]: Just checked, it looks like we call `disableFeature` if a user downgrades from business.

Comment (anafilipadealmeida) [reply packages/composer/composer/components/ShopgridComposerBar.jsx:67]: `hasShopgridFlip` is no longer coming from the feature flips, it's the `hasGridFeature` from the org parser. As a note, there's another task for later (https://buffer.atlassian.net/browse/PUB-3006) to replace orgs data we currently get from profiles, and the `shouldShowGridPreview` is one of those cases 🙂

**Review by hamstu — APPROVED** [review]
LGTM!

**Review by albennett — APPROVED** [review]
Great clean up, Ana!

**Pull request merged**